samples/basic/blinky_pwm/README.rst

```diff
@@ -53,6 +53,8 @@ In these other cases, however, manual wiring is necessary:
      - connect PWM1 (PA8) to an LED
    * - :zephyr:board:`nucleo_wb55rg`
      - connect PWM1 (PA8) to an LED
+   * - :zephyr:board:`nucleo_wba55cg`
+     - connect PWM2 (PA9) to an LED
```

Comment:
I'm confused, isn't the LED already populated on the board, hence why you were asked to move its definition to the board devicetree? Or is there no such LED on the board, in which case that request was wrong? :/